spec/models/provider_relationship_permissions_spec.rb

```diff
@@ -95,4 +95,59 @@
       expect(described_class.all_relationships_for_providers([training_provider, ratifying_provider])).to eq([relationship])
     end
   end
+
+  describe '#partner_organisation' do
+    let(:training_provider) { create(:provider) }
+    let(:ratifying_provider) { create(:provider) }
+    let!(:relationship) do
+      create(
+        :provider_relationship_permissions,
+        training_provider: training_provider,
+        ratifying_provider: ratifying_provider,
+      )
+    end
+
+    context 'for a user from ratifying provider' do
+      let(:provider_user) { create(:provider_user, providers: [ratifying_provider]) }
+
+      it 'is the training provider' do
+        expect(relationship.partner_organisation(provider_user)).to eq(training_provider)
+      end
+    end
+
+    context 'for a user from training provider' do
```

Comment:
Can we test what happens if you belong to both orgs too?

Reply:
https://github.com/DFE-Digital/apply-for-teacher-training/pull/5235/commits/07b1b0a3903d892eff3bc03985e03eb3a5c44318 was a bit more complex than I first anticipated, hopefully test scenarios cover this well now.